packages/auto-router/src/lib/auto-router.ts

```diff
@@ -322,7 +320,7 @@ export class AutoRouter {
 		const [zeroExSwap, anatomy, inactiveAnatomy, amounts] = await Promise.all([
 			this.zeroExAggregator.quote(
 				index.address,
-				outputToken.address,
+				outputTokenAddress ?? 'ETH',
```

Comment:
do we still need to have `outputToken` function param as an ERC20 instead of an address?